src/content/docs/synthetics/synthetic-monitoring/administration/synthetic-public-minion-ips.mdx

```diff
@@ -14,9 +14,11 @@ redirects:
   - /docs/synthetics/synthetic-public-minion-ips
 ---
 
-New Relic uses a group of minions to execute your [synthetic monitors](/docs/synthetics/new-relic-synthetics/using-monitors/add-edit-monitors). These minions are deployed in different data centers around the globe, and they are in charge of actually running your monitors.
+You can set up minions to execute your [synthetic monitors](/docs/synthetics/new-relic-synthetics/using-monitors/add-edit-monitors). These minions are deployed from different data centers around the globe and are in charge of running your monitors.
```

Comment:
```suggestion
You can set up minions to execute your [synthetic monitors](/docs/synthetics/new-relic-synthetics/using-monitors/add-edit-monitors). These minions are deployed from different data centers around the globe and run your monitors.
```

Reply:
How do we feel about: 

> You can set up minions to execute your [synthetic monitors](/docs/synthetics/new-relic-synthetics/using-monitors/add-edit-monitors). These minions run your monitors and are deployed from different data centers around the globe.

Reply:
The original wording is less confusing to me. When we mention that customers can set up minions it will cause confusion between public and private minions. We create and maintain minions for our public locations where customers are responsible for deploying minions only for private locations behind the firewall.

```suggestion
New Relic maintains public minions to execute your [synthetic monitors](/docs/synthetics/new-relic-synthetics/using-monitors/add-edit-monitors). These minions are deployed in different data centers around the globe and are in charge of running your monitors.
```